### DISCUSSION TITLE
How to work with Ferrum::DeadBrowserError

### DISCUSSION THREAD

**Discussion by davidwessman** [rubycdp/ferrum#397] [General]
Hello!

We are getting some `Ferrum::DeadBrowserError` which I have a hard time wrapping my head around.

The only reference I find to the documentation is the `ws_max_receive_size`, but is that the only time we get DeadBrowserError back? Would it make sense to increase the 64 MB default?

**Comment by route** [comment]
@davidwessman did you check RAM? No this param is not for that purpose

**Reply by davidwessman** [reply]
Yeah, memory usage spiking making Chrome crash is probably the reason.
Thank you 🙂
